CountdownWidget/CountdownWidget/src/CountdownResult.ts

```diff
@@ -36,6 +36,18 @@ export class CountdownResult {
 	public getDisplayUnit(): string {
 		return Unit[this.unit].toLowerCase();
 	}
+
+	public isLessThan(threshold: number, unit: Unit) {
+		if (this.unit < unit) {
+			return true;
+		}
+
+		if (this.unit === unit) {
+			return this.value < threshold;
+		}
+
+		return false;
+	}
 }
 
-export enum Unit { Years, Days, Hours, Minutes, Seconds, Invalid }
+export enum Unit { Invalid = 0, Seconds = 1, Minutes = 2, Hours = 3, Days = 4, Years = 5}
```

Comment:
Very nice, much clearer to make this explicit I think. 
However, does it make any difference in how it has been used up to this point? Is there any places where > < is being used to compare between them?

Reply:
Definitely like how easy it is to read the code. Are there any places where this.value can be large enough to be a different unit (for example, 90 minutes)?